docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -574,191 +574,35 @@ used to delimit variable names.
   doubling it. For example, to emit a header with the value ``100%``, the custom header value in
   the Envoy configuration must be ``100%%``.
 
-Supported variable names are:
+All HTTP :ref:`Command Operators <config_access_log_command_operators>` used for access logging may be specified
+in Custom request/response headers. However, depending where a particular Command Operator is used, the context needed for
+the Operator may not be available and the produced output is empty string. For example, the following configuration
+uses ``%RESPONSE_CODE%`` operator to modify request headers using code from the response.
+The output is an empty string, because request headers are modified
+before the request is sent upstream and the response is not received yet.
 
-%DOWNSTREAM_REMOTE_ADDRESS%
-    Remote address of the downstream connection. If the address is an IP address it includes both
-    address and port.
+.. code-block:: yaml
```

Comment:
it would be better to move this to a separate yaml file with a full boostrap config - that way it would be tested/validated and wont bitrot

Reply:
Good idea. Is there an example how to do that?

Reply:
if you `git grep literalinclude docs` you should find other examples

Reply:
Added full config. Thanks for the idea!